### PR TITLE
indent option

### PR DESCRIPTION
**Pull request by rap2hpoutre**
Added indent option. Fixes https://github.com/jcarpanelli/spinnies/issues/14

Capture d’écran 2019-07-24 à 11 12 46

### CODE REVIEW

**Comment by SweetMNM** [comment]
If you try to indent a multi-line text spinner everything breaks.
Try using the `indent` option on the spinner _that would love to make some friends_ and see for yourself.
The problem is getLinesLength and breakText don't take the `indent` option into consideration.

Also multi-line strings should have all lines indented.

**Comment by jcarpanelli** [comment]
Hello, @rap2hpoutre! Sorry for the super late response! I just came back from vacations! I'll review this PR at the end of the weekend :slightly_smiling_face:.

**Comment by rap2hpoutre** [comment]
I fixed the multiline problem thanks to @SweetMNM comment (in ac1a4b0)

@jcarpanelli You could now retry, it seems to work for me (even with multiline).

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ Parameters:
 - **name** - `string`: spinner reference name.
 - **options** - `object`:
   - **text**: - `string`: Optional text to show in the spinner. If none is provided, the `name` field will be shown.
+  - **indent**: - `number`: Optional, indent the spinner with the given number of spaces.
```
Comment (jcarpanelli) [inline README.md]: :raised_hands:

Comment (rap2hpoutre) [reply README.md]: :)

**Comment by rap2hpoutre** [comment]
@jcarpanelli I added some tests in this commit: d515657. Let me know if it' OK for you

**Comment by rap2hpoutre** [comment]
@jcarpanelli 👍 Thank you! When do you plan to release it?

**Comment by jcarpanelli** [comment]
Hey @rap2hpoutre! I've just published `v0.5.0` :slightly_smiling_face:

**Comment by rap2hpoutre** [comment]
Thank you!!

**Pull request merged**